fix installation with pip by using setuptools.{install,setup} rather than distutils.*
author: Kenneth Hoste (HPC-UGent)
loosely inspired by https://bitbucket.org/scons/scons/pull-requests/113
cfr. https://github.com/SCons/scons/issues/2769
--- scons-3.0.4/setup.py.orig	2019-01-23 18:30:57.000000000 +0100
+++ scons-3.0.4/setup.py	2019-02-27 14:07:50.139560710 +0100
@@ -27,9 +27,8 @@
 import distutils.command.install_scripts
 import distutils.command.install_lib
 import distutils.command.install_data
-import distutils.command.install
-import distutils.core
-import distutils
+import setuptools.command.install
+import setuptools
 # import setuptools
 """
 NOTE: Installed SCons is not importable like usual Python packages. It is
@@ -77,7 +76,7 @@
     is_win32 = 1
 
 
-_install = distutils.command.install.install
+_install = setuptools.command.install.install
 _install_data = distutils.command.install_data.install_data
 _install_lib = distutils.command.install_lib.install_lib
 _install_scripts = distutils.command.install_scripts.install_scripts
@@ -510,7 +509,7 @@
                  'build_scripts': build_scripts},
 }
 
-distutils.core.setup(**arguments)
+setuptools.setup(**arguments)
 
 if Installed:
     for i in Installed:
